openhcl/virt_mshv_vtl/src/processor/snp/mod.rs

```diff
@@ -420,16 +420,19 @@ impl BackingPrivate for SnpBacked {
             .lapic
             .scan(&mut this.vmtime, scan_irr);
 
-        if nmi {
-            this.handle_nmi(vtl);
-        }
+        // Interrupts are ignored while waiting for SIPI.
```

Comment:
I wonder if we should check for incoming SIPI (the check below) before we throw the NMI away.

Reply:
Hmmmm. I guess? I don't think it matters too much, what are the odds?

Reply:
Odds do seem low, I agree. But why not just order the code to close the gap?